tests/dialects/test_singlestore.py

```diff
@@ -560,6 +560,21 @@ def test_time_functions(self):
                 "singlestore": "SELECT DATEDIFF('2009-12-31', '2009-01-01') AS numweeks",
             },
         )
+        self.validate_all(
+            "SELECT CURRENT_DATE()",
+            read={
+                "": "SELECT CURRENT_DATE()",
+                "singlestore": "SELECT CURRENT_DATE",
+            },
+        )
+        self.validate_all(
+            "SELECT UTC_DATE()",
```

Comment:
Is this form equivalent to the `UTC_DATE` one (i.e., no parentheses)?

Reply:
Yep, they are the same.
```
MySQL [db]> SELECT UTC_DATE();
+------------+
| UTC_DATE() |
+------------+
| 2025-09-03 |
+------------+
1 row in set (0.001 sec)

MySQL [db]> SELECT UTC_DATE;
+------------+
| UTC_DATE   |
+------------+
| 2025-09-03 |
+------------+
1 row in set (0.001 sec)

```